packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -67,6 +67,8 @@ type AuthDataMap = Record<
     }
 >
 
+const isStorefrontPreview = isOriginTrusted(getParentOrigin())
```

Comment:
Couple comments here. Firstly, Alex make a good comment about having Preview "leak" in to the commerce-sdk-react library when it doesn't need to know anything about it. So we might want to come up with another name. Secondly, I think maybe this will return "true" when you are on development, we might want to double check that. 

The variable name could be something like `inTrustedIframe`. We'll want to pay special attention how this all works in development tho.

Reply:
Done. I renamed the variable `isParentTrusted` and the cookie key to use the suffix '-iframe' instead of '-preview'. As we traced the logic, we discovered that when the storefront loads on the local development server 'localhost:3000', the returned value is false."

As we saw doing a mental trace of the logic, when the storefront loads on the local development server 'localhost:3000', the returned value is false.